### PR TITLE
Infinite scroll

### PR DESCRIPTION
**Pull request by hosseintoussi**

### CODE REVIEW

**Comment by hosseintoussi** [comment]
one pr for all, and closing the previous one. @aaronmyatt

File: app/views/cards/_header.html.erb [hunk 1]
```diff
@@ -13,22 +13,25 @@
           <% end %>
         </div>
         <div class='ui item'>
-          <img class="ui avatar image" src="<%= avatar_url(current_user) %>">
-          <span><%= current_user.email %></span>
+          <span><%= link_to user_path(current_user), class: 'ui item' do %>
+            <img class="ui avatar image" src="<%= avatar_url(current_user) %>">
```
Comment (aaronmyatt) [inline app/views/cards/_header.html.erb]: Holy shit, didn't know I could use blocks to wrap links around things like this ;o

Comment (hosseintoussi) [inline app/views/cards/_header.html.erb]: 💃 👯

File: app/views/cards/_header.html.erb [hunk 2]
```diff
@@ -23,9 +23,9 @@
     <div class="ui right dropdown item">
       <i class="sidebar icon"></i>
       <div class="menu">
-        <%= link_to user_path(current_user), class: 'ui item' do %>
-          <i class="user icon"></i>
-          Profile
```
Comment (aaronmyatt) [inline app/views/cards/_header.html.erb]: I think we should still have this link to Profile.

Comment (hosseintoussi) [inline app/views/cards/_header.html.erb]: profile link is now on user email, and profile pic, something like fb

**Comment by aaronmyatt** [comment]
Did you test the infinite scroll

**Pull request merged**